PluginInstaller/Create-PluginInstaller.ps1

```diff
@@ -122,18 +79,19 @@ function main {
     }
 
     function prep-template {
-      $contents = $template
-
-      if ( $upgrade_code -match "$" ) { $upgrade_code = Invoke-Expression ($upgrade_code) }
-
-      $contents = $contents.Replace($default_title_cab, $title.Replace(" ", "_"));
-      $contents = $contents.Replace($default_title, $title);
-      $contents = $contents.Replace($default_manufacturer, $manufacturer);
-      $contents = $contents.Replace($default_description, $description);
-      $contents = $contents.Replace($default_product_version, $product_version);
-      $contents = $contents.Replace($default_upgrade_code, $upgrade_code);
-      $contents = $contents.Replace($default_product_code, $product_code);
-      $contents = $contents.Replace($default_version_short, $version_short);
+      $contents = $templateXml
+
+      $contents = $contents.Replace("%MANUFACTURER_NAME%", $manufacturer_name);
```

Comment:
If these all come from the branding, why use an env var at all?
Why not just use @MANUFACTURER_NAME@ and rely on the makefile to do the replacements?